tools/Azure.Mcp.Tools.Foundry/src/Commands/ModelDeploymentCommand.cs

```diff
@@ -13,16 +15,6 @@ namespace Azure.Mcp.Tools.Foundry.Commands;
 public sealed class ModelDeploymentCommand : SubscriptionCommand<ModelDeploymentOptions>
 {
     private const string CommandTitle = "Deploy Model to Azure AI Services";
-    private readonly Option<string> _deploymentNameOption = FoundryOptionDefinitions.DeploymentNameOption;
-    private readonly Option<string> _modelNameOption = FoundryOptionDefinitions.ModelNameOption;
-    private readonly Option<string> _modelFormatOption = FoundryOptionDefinitions.ModelFormatOption;
-    private readonly Option<string> _azureAiServicesNameOption = FoundryOptionDefinitions.AzureAiServicesNameOption;
-    private readonly Option<string> _modelVersionOption = FoundryOptionDefinitions.ModelVersionOption;
-    private readonly Option<string> _modelSourceOption = FoundryOptionDefinitions.ModelSourceOption;
-    private readonly Option<string> _skuNameOption = FoundryOptionDefinitions.SkuNameOption;
-    private readonly Option<int> _skuCapacityOption = FoundryOptionDefinitions.SkuCapacityOption;
-    private readonly Option<string> _scaleTypeOption = FoundryOptionDefinitions.ScaleTypeOption;
-    private readonly Option<int> _scaleCapacityOption = FoundryOptionDefinitions.ScaleCapacityOption;
```

Comment:
Thanks for cleaning this up, never really liked this pattern as it really didn't serve much of a purpose.

Probably should update `new-command.md` to have a guideline like

`Use *OptionDefinition Options directly in RegisterOptions and BindOptions`